feign-reactor-core/src/main/java/reactivefeign/client/statushandler/ReactiveStatusHandlers.java

```diff
@@ -57,7 +57,8 @@ private static Response buildFeignResponseForDecoder(ReactiveHttpResponse respon
     Request feignRequest = Request.create(httpMethod(request.method()),
             request.uri().toString(),
             (Map)request.headers(),
-            Request.Body.empty());
+            Request.Body.empty(),
+            request.requestTemplate());
```

Comment:
I'd like to avoid exposing `requestTemplate()` in reactive request. Do we really need this data or may try to use some stub like with `Request.Body.empty()`?

Reply:
OK.

Looks like safe to just pass through null, like the other deprecated static create method does